static/css/components/observationStats.less

```
@@ -73,15 +73,16 @@
 }
 
 .login-link, .stats-link {
-  margin: 0 10px;
-
   a {
     text-decoration: none;
   }
 }
 
-.stats-link {
-  margin-top: 14px;
+.review-cta {
+  margin: 0 10px;
+  .stats-link {
+    margin-top: 14px;
+  }
```

```suggestion
.review-cta {
  margin: 14px 10px 0;
```

Ensures that login link has a top margin, as well.